www/scripts/productlist/ProductSettingsView.js

```diff
@@ -69,7 +69,7 @@ function (declare, domAttr, domClass, domConstruct, Dialog, Button,
     0 : ['submit'],
     /* PRODUCT_ADMIN */ 1 : ['name', 'description'],
     /* SUPERUSER */     2 : ['endpoint', 'dbengine', 'dbhost', 'dbport',
-                             'dbuser', 'dbpass', 'dbname']
+                             'dbuser', 'dbpass', 'dbname', 'runlimit']
```

Comment:
#1410 states that `product admin`s should be able to change this variable too, not just superusers.

Reply:
@whisperity If I remember it correctly we discussed that only the superuser should  have the right to change this.

Reply:
Sure. :wink: I don't think I was there on that discussion. But it sounds good that it's a superuser thing.